extension/intents/self/contentScript.js

```diff
@@ -0,0 +1,45 @@
+/* globals communicate, Fuse */
+
+communicate.register("focusField", async message => {
+  const helpElement = message.query;
+  const elements = [];
+  const searchContent = [];
+  for (const el of document.querySelectorAll(".intent,.description")) {

```

Comment:
This should include `.utterance` too – it will match something like "scroll" then (which is in a `<div class=utterance>`)